src/com/aca/narekhakobyan/homework/employee/SellManager.java

```diff
@@ -0,0 +1,283 @@
+                    stockAMD -= gadget.getPrice() * count;
+                    logger.info(gadget.toString());
+                    return;
+                } else {
+                    Logger logger = Logger.getLogger("Errors");
+                    FileHandler errorsLog = null;
+                    try {
+                        errorsLog = new FileHandler("src\\com\\aca\\narekhakobyan\\homework\\errors\\errors.log", true);
+                        logger.addHandler(errorsLog);
+                        logger.severe("Sorry. There are only %d items.\n" + gadget.getCount());
+                    }
+                    catch (IOException e){
+                        e.printStackTrace();
+                    }
+                }
+            }
+        }
+    }
+
+    @Override
+    public void sellItem(String name) {
+        sellItem(name, 1);
+    }
+
+    public void addItem(Object gadgetObject) {
+        Pair<Boolean, Integer> isGadgetExistVar = isGadgetExist(gadgetObject);
+        if (isGadgetExistVar.getKey()) {
+            Integer tempCount = gadgets.get(isGadgetExistVar.getValue()).getCount();
+            gadgets.get(isGadgetExistVar.getValue()).setCount(tempCount + ((Gadget)gadgetObject).getCount());
+            stockAMD += gadgets.get(isGadgetExistVar.getValue()).getPrice() * tempCount;
+        } else {
+            gadgets.add((Gadget) gadgetObject);
+            stockAMD += ((Gadget) gadgetObject).getPrice() * ((Gadget) gadgetObject).getCount();
+        }
+    }
+
+    private Pair<Boolean, Integer> isGadgetExist(Object gadgetObject) {
+        Pair<Boolean, Integer> retVal;
+        int index = -1;
+        for (Gadget gadget : gadgets) {
+            index++;
+            if (gadget.equals(gadgetObject)) {
+                retVal = new Pair<>(Boolean.TRUE, index);
+                return retVal;
+            }
+        }
+        retVal = new Pair<>(Boolean.FALSE, index);
+        return retVal;
+    }
+
+    private boolean isCorrectPrice(Integer price) {
+        return price >= 0;
+    }
+
+    @Override
+    public void changePrice(Object gadgetObject, Integer price) {
+        for (Gadget gadget : gadgets) {
+            if (gadget.getClass().equals(gadgetObject.getClass())) {
+                if (price < 0) {
+                    Logger logger = Logger.getLogger("Errors");
+                    FileHandler errorsLog = null;
+                    try {
+                        errorsLog = new FileHandler("src\\com\\aca\\narekhakobyan\\homework\\errors\\errors.log", true);
+                        logger.addHandler(errorsLog);
+                        logger.severe("Incorrect price. Price should be non-negative");
+                    }
+                    catch (IOException e){
+                        e.printStackTrace();
+                    }
+                }
+                stockAMD += gadget.getCount() * (price - gadget.getPrice());
+
+            }
+        }
+    }
+
+    public void turnOn() {
```

Comment:
please review this requirement Nar jan _turn on gadget - it means press playVideoRecording or playAudioRecording and 
and if gadget supports both of them so press videorecording then audiorecording_

move this functionality into gadget model, and from here just call gadget.turnOn()